build_calicoctl/create_binary.sh

```diff
@@ -1,14 +1,32 @@
 #!/bin/bash
 
 set -e
+set -x
 
 docker build -t calico-build .
 
 cd ..
 mkdir -p `pwd`/dist
 chmod 777 `pwd`/dist
-docker run -u user -v `pwd`/:/code/calico -v `pwd`/dist:/code/dist calico-build pyinstaller calico/calicoctl.py -a -F -s --clean
-docker run -u user -v `pwd`/dist:/code/dist calico-build bash -c 'docopt-completion --manual-bash dist/calicoctl && mv calicoctl.sh dist'
+
+if [[ $CIRCLE_TEST_REPORTS ]]; then
```

Comment:
Yeah, I don't like it either. I'd like to be able to run the UTs locally. So I need a way to say "If you're running this on circleci, go ahead and pass the coverage args to nose. Otherwise don't try." I could do that by putting the call to nose back in circle.yml, but I thought we wanted to fold the UTs into the build pipeline.